### PR TITLE
ion.RangeSlider definitions matching npm package name

### PR DESCRIPTION
**Pull request by Karel-van-de-Plassche**
The current ion.RangeSlider definitions do not match the name of the package in NPM. This pull request fixes this, and adds some updates the definitions.

New pull request without the settings parse files based on #25577

Note: `npm test` currently fails on another package. edit: Seems Travis runs it fine though!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If removing a declaration:
- [x] Delete the package's directory.

### CODE REVIEW

**Comment by sixinli** [comment]
looking..

File: types/ion-rangeslider/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    interface JQuery {
+        destroy(): void;
+        ionRangeSlider(options?: IonRangeSliderOptions): JQuery;
+        reset(): void;
+        update(option: IonRangeSliderOptions): void;
+    }
+}
+
+export interface IonRangeSliderOptions {
+    type?: string;    // Choose slider type, could be `single` - for one handle, or `double` for two handles [Default: single]
```
Comment (sixinli) [inline types/ion-rangeslider/index.d.ts]: can you alph these...? kind of hard to compare whats changed

Comment (Karel-van-de-Plassche) [reply types/ion-rangeslider/index.d.ts]: I'm not sure what you mean. For this file I have:
- Added `declare global { }` enclosure
- Updated the comments in the file header
- Added comment strings after each argument (which I prefer, but I'm not sure about the TS crowd)

Comment (minestarks) [reply types/ion-rangeslider/index.d.ts]: Git didn't detect the rename because the file changed too much, presumably. Changes look OK to me.

Comment (sixinli) [reply types/ion-rangeslider/index.d.ts]: its easier to search and browse through the list... when there's no easy functional grouping alph is preferred IMO :)

Comment (minestarks) [reply types/ion-rangeslider/index.d.ts]: Fair point. If you want to open a PR to alphabetize these I would approve it.

Comment (Karel-van-de-Plassche) [reply types/ion-rangeslider/index.d.ts]: Right now I conserved the order from the documentation site of the library (http://ionden.com/a/plugins/ion.rangeSlider/en.html) which has some loose ordering based on functionality. If alphabetical is preferred, I don't mind opening a new pull request

Comment (Karel-van-de-Plassche) [reply types/ion-rangeslider/index.d.ts]: Right now I conserved the order from the documentation site of the library (http://ionden.com/a/plugins/ion.rangeSlider/en.html) which has some loose ordering based on functionality. If alphabetical is preferred, I don't mind opening a new pull request

**Pull request merged**